src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -847,42 +848,134 @@ private void RebuildRegionConnections()
         foreach (var connection in regionConnectionLines.Values)
             connection.Free();
 
+        foreach (var connection in additionalConnectionLines)
+            connection.Free();
+
         regionConnectionLines.Clear();
+        additionalConnectionLines.Clear();
 
         foreach (var entry in connections)
         {
             var region1 = map.Regions[entry.Key.x];
             var region2 = map.Regions[entry.Key.y];
 
-            var region1Shown = region1.Visibility == MapElementVisibility.Shown;
-            var region2Shown = region2.Visibility == MapElementVisibility.Shown;
+            var vis1 = region1.Visibility;
```

Comment:
```suggestion
            var visibility1 = region1.Visibility;
```

`vis` is not on the approved list of abbreviations so it shouldn't be used.